cuda_core/cuda/core/experimental/_memory.pyx

```diff
@@ -25,11 +25,15 @@ import weakref
 from cuda.core.experimental._dlpack import DLDeviceType, make_py_capsule
 from cuda.core.experimental._stream import Stream, default_stream
 from cuda.core.experimental._utils.cuda_utils import ( driver, Transaction, get_binding_version, 
-                                                       wrap_driver_function_with_error_handling,
+                                                       handle_return,
                                                      )
 
 if platform.system() == "Linux":
     import socket
+# Helper to invoke CUDA driver functions with standard error handling.
+def _driver_call(func, /, *args):
+    handle_return(func(*args))
```

Comment:
Does this abstraction add value over just using `handle_return` directly?

We also already import `_check_driver_error as raise_if_driver_error` which if we don't need return values from the driver calls then that will be more efficient. Apologies for pointing towards `handle_return` instead of the driver specific function.

Reply:
Done